refs/legacy.json

```diff
@@ -89,7 +89,7 @@
     "SP800-67": "<a href=\"https://csrc.nist.gov/publications/nistpubs/800-67-Rev1/SP-800-67-Rev1.pdf\"><cite> Recommendation for the Triple Data Encryption Algorithm (TDEA) Block Cipher, Revised January 2012.</cite></a> SP-800-67 Revision 1. U.S. Department of Commerce/National Institute of Standards and Technology. URL: <a href=\"https://csrc.nist.gov/publications/nistpubs/800-67-Rev1/SP-800-67-Rev1.pdf\">https://csrc.nist.gov/publications/nistpubs/800-67-Rev1/SP-800-67-Rev1.pdf</a>",
     "SUITEB": "<a href=\"http://www.nsa.gov/ia/programs/suiteb_cryptography/\"><cite>NSA Suite B Cryptography</cite></a>. URL: <a href=\"http://www.nsa.gov/ia/programs/suiteb_cryptography/\">http://www.nsa.gov/ia/programs/suiteb_cryptography/</a> ",
     "SUS3": "Cathy Hughes. <a href=\"http://www.unix.org/single_unix_specification/\"><cite>The Single UNIX Specification, Version 3</cite></a>. 2004 URL: <a href=\"http://www.unix.org/single_unix_specification/\">http://www.unix.org/single_unix_specification/</a>",
-    "THEGRID": "Allen Hurlburt. <cite>Grid: A Modular System for the Design and Production of Newpapers, Magazines, and Books.</cite> Wiley. 1982. ISBN 0-471-28923-X.",
+    "THEGRID": "Allen Hurlburt. <cite>Grid: A Modular System for the Design and Production of Newspapers, Magazines, and Books.</cite> Wiley. 1982. ISBN 0-471-28923-X.",
```

Comment:
Some objects appear on the web in multiple flavors. This book is available with both spellings from Amazon, but the correct spelling can be seen on various places, including, https://www.si.edu/object/siris_sil_105907